common/src/View/SetBrushFaceAttributesTool.cpp

```diff
@@ -183,7 +183,7 @@ namespace TrenchBroom {
                 return true;
             }
 
-            if (!m_dragTargetFaceHandle && !m_dragTargetFaceHandle) {
```

Comment:
Skimming over the rest of the file, I think they're either both `std::nullopt`, or both non-null. So it's just a cosmetic problem, which is good.